tests-compat/compat.spec.ts

```diff
@@ -130,4 +130,18 @@ describe('@vue/compat build', () => {
 
     expect(wrapper.html()).toBe('<div>stubbed</div>')
   })
+
+  it('wrapper.vm points to correct instance when component is wrapped with Vue.extend', () => {
+    const Component = extend({
+      data() {
+        return { foo: 'bar' }
+      },
+      template: '<div></div>'
+    })
+
+    debugger
```

Comment:
@xanf , Probably, you forgot to remove the debugger statement

Reply:
Lol. thank you for catching!